src/pivot-table/components/cells/ListCellFactory.tsx

```diff
@@ -72,7 +72,7 @@ const ListCellFactory = ({ index, style, data }: ListCallbackProps): JSX.Element
     <DimensionCell
       cell={cell}
       data={data}
-      rowIndex={cell.y}
+      rowIndex={cell.pageY}
```

Comment:
Just noticed, for another refactor PR we could remove `rowIndex` and `colIndex` props. The data is already available in the `cell` prop. That would make it more obvious what kind of positional data it is in the DimensionCell component.

Reply:
I'll create a refactor ticket for it 👍